### PR TITLE
Fix #4262 - remove service

### LINKED ISSUE #4262: Example 2: Creates a service?
In Example 2, you say "This command creates a service named TestService."

Are you sure it CREATES a service?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e4febd11-ae9e-4b34-ca70-5929ee1222fa
* Version Independent ID: fb28372c-479d-c3bd-8f8a-f8c5523ac5cf
* Content: [Remove-Service (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/remove-service?view=powershell-6#feedback)
* Content Source: [reference/6/Microsoft.PowerShell.Management/Remove-Service.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Remove-Service.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[6349e08](https://github.com/sdwheeler/PowerShell-Docs/pull/4269/commits/6349e0820e8537f264b5967491d07c64a81215b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Remove-Service.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4262/reference/6/Microsoft.PowerShell.Management/Remove-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/remove-service?view=powershell-6&branch=pr-en-us-4269) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905061757278392-4269%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OWW0U4HAFBNySBlB6UMHpN8mvI%2BbxvTe7MsgQBczkvM%3D&st=2019-05-06T17%3A56%3A27Z&se=2019-06-06T18%3A01%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md [hunk 1]
```diff
@@ -29,34 +29,34 @@ Remove-Service [-InputObject <ServiceController>] [-WhatIf] [-Confirm] [<CommonP
...
+The `Remove-Service` cmdlet removes a Windows service in the registry and in the service database.
 
 ## EXAMPLES
 
 ### Example 1: Remove a service
 
-```
-PS C:\> Remove-Service -Name "TestService"
-```
+This removes removes a service named TestService.
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Double words.

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md [hunk 2]
```diff
@@ -72,8 +72,7 @@ Accept wildcard characters: False
 
 ### -Name
 
-Specifies the service names of the services to stop.
-Wildcard characters are permitted.
+Specifies the service names of the services to stop. Wildcard characters are permitted.
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Stop seems wrong, should be remove.

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md [hunk 3]
```diff
@@ -138,7 +138,8 @@ This cmdlet does not return any output.
 
 ## NOTES
 
-* To run this cmdlet on Windows Vista and later versions of the Windows operating system, start PowerShell by using the Run as administrator option.
+To run this cmdlet on Windows Vista and later versions of the Windows operating system, start
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Vista isn't listed as supported for PS Core. Do we need this note still?

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md (hunk 1 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Fixed

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md (hunk 2 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Fixed

File: reference/6/Microsoft.PowerShell.Management/Remove-Service.md (hunk 3 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Management/Remove-Service.md]: Fixed

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[0d89013](https://github.com/sdwheeler/PowerShell-Docs/pull/4269/commits/0d89013d92fd99778974ac64bf94efeee60677cb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Remove-Service.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4262/reference/6/Microsoft.PowerShell.Management/Remove-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/remove-service?view=powershell-6&branch=pr-en-us-4269) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905061953003320-4269%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=AzHoBvZQO7VlQg1HEBxxaU6%2F2ShbxfMgxnprWLZRZag%3D&st=2019-05-06T19%3A52%3A03Z&se=2019-06-06T19%3A57%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[16b5172](https://github.com/sdwheeler/PowerShell-Docs/pull/4269/commits/16b5172356be0e8799f3bda535499245d1b024a6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Remove-Service.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4262/reference/6/Microsoft.PowerShell.Management/Remove-Service.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/remove-service?view=powershell-6&branch=pr-en-us-4269) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905061955062008-4269%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=98WF5k4wFFDf5IvIsnOijcbIqEkYgNjfYrCTd4DoC4E%3D&st=2019-05-06T19%3A56%3A00Z&se=2019-06-06T20%3A01%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**